packages/companies/companies.txt

```diff
@@ -1,86 +1,166 @@
 2GIS
+3M
 3M Company
+A.O. Smith Corp
 A.P. Moller-Maersk Group
+AAPL
 ABB
 Abbott Laboratories
+Abbott Laboratories
```

Comment:
Looks like there might be a bunch of dupes from this

Reply:
Thanks, I missed that.

Reply:
Might make sense to strip punctuation if you're doing this through a script

Reply:
A few duplicates won't hurt. They are removed during the preprocessing.